bukkit/src/main/java/com/leonardobishop/quests/bukkit/hook/versionspecific/VersionSpecificHandler9.java

```diff
@@ -15,7 +19,22 @@ public boolean isPlayerGliding(Player player) {
     }
 
     @Override
-    public boolean isPlayerOnStrider(Player player) {
-        return false;
+    public int getAvailableSpace(Player player, ItemStack newItemStack) {
+        int availableSpace = 0;
+        PlayerInventory inventory = player.getInventory();
+        HashMap<Integer, ? extends ItemStack> itemStacksWithSameMaterial = inventory.all(newItemStack.getType());
+        for (ItemStack existingItemStack : itemStacksWithSameMaterial.values()) {
+            if (newItemStack.isSimilar(existingItemStack)) {
+                availableSpace += (newItemStack.getMaxStackSize() - existingItemStack.getAmount());
+            }
+        }
+
+        for (ItemStack existingItemStack : inventory.getStorageContents()) {
```

Comment:
The difference for `VersionSpecificHandler#getAvailableSpace` between 8 and 9 is that 8 has `PlayerInventory#getContents` instead of `PlayerInventory#getStorageContents`